numba/targets/arraymath.py

```diff
@@ -3948,3 +3948,89 @@ def np_kaiser_impl(M, beta):
         return _i0n(n, alpha, beta)
 
     return np_kaiser_impl
+
+
+@register_jitable
+def cross_operation(a, b, cp):
+    dta = a.dtype
+    a0 = a[..., 0]
+    a1 = a[..., 1]
+    if a.shape[-1] == 3:
+        a2 = a[..., 2]
+    else:
+        a2 = np.multiply(dta.type(0), a0)
```

Comment:
Trying this desperate type juggling to solve the Windows build failures: https://dev.azure.com/numba/numba/_build/results?buildId=2424&view=logs

Reply:
It worked! 😄